### ISSUE TITLE
vere v1.4: address 0x18 out of loom!

### ISSUE THREAD

**Issue by tinnus-napbus** [urbit/urbit#4827]
**Describe the bug**
This has happened twice now since upgrading to vere v1.4:

```
~tinnus-napbus:dojo> address 0x18 out of loom!
loom: [0x200000000 : 0x280000000)
Assertion '0' failed in noun/events.c:129
                                         home: bailing out                                    
serf: pier unexpectedly shut down
Aborted
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Upgrade to vere v1.4
2. ???
3. Profit

**Expected behaviour**
Should not crash

**System (please supply the following information, if relevant):**
 - OS: ubuntu 20.04
 - Vere and Urbit OS versions: v1.4 / hsgqe
 - Your ship's `%base` hash 0vc.qjue1.o13n2.a68b4.lt74p.n4qfk.u3roo.uss4q.2i9m3.68cid.hsgqe

**Comment by botter-nidnul** [comment]
Happens on first boot, every time I've tried it so far.

**Comment by philipcmonk** [comment]
This seems to have been introduced by #4766.  Specifically, in `_cttp_creq_connect` we set the hostname for TLS handshake before creating the connection (vs after before this PR), and I guess that means `ceq_u->cli_u` may be null when we try to access it.

**Comment by joemfb** [comment]
That's right. Fixed in #4828. My apologies!
